app/js/views/element_call_button_view.js

```diff
@@ -0,0 +1,54 @@
+import { View } from 'components/fxos-mvc/dist/mvc';
+import 'components/gaia-button/gaia-button';
+
+(function(window) {
+'use strict';
+
+var shadowTemplate =
+`<style scoped>
+gaia-button {
+	--button-background: green;
+	--button-background-active: lightgreen;
+	--button-color: white;
+	--button-color-active: white;
+	width: 10rem;
+	min-width: 0 !important;
+	margin: 0.5rem !important;
+}
+</style>
+<gaia-button>Call</gaia-button>`;
+
+var proto;
+
+export default class ElementCallButtonView extends View {
+	constructor(controller, phoneFieldController) {
+		proto = Object.create(HTMLElement.prototype);
+		proto.createdCallback = this._createdCallback;
+		proto._scope = this;
```

Comment:
I think it's debatable whether custom elements can be the "V" in MVC. However, currently, the base view class in mvc.js feels like a much "lighter" construct for composing views. I agree though that this custom element isn't using proper architecture here. If you look at my PR for fxos/camera you will see I use both views and custom elements for different things. My "CaptureView" is a lightweight construct for composing several related view elements in a single class that can be glued to a controller. However, I also have a FXOSCamera custom element that is used to abstract the complexity of positioning the viewfinder. CaptureView includes an FXOSCamera custom element in its template along with a few button elements. Personally, I feel this is the simplest approach and gives us the advantages of both custom elements as well as a lightweight view class.